wiki/Article_Styling_Criteria/en.md

```diff
@@ -108,55 +114,59 @@ Folder names must not contain URL reserved characters. Generally, all folder nam
+Folder names must not use characters that require percent encoding. Those characters may include, but are not limited to:
 
 - spaces
-  - all spaces must be underscores (`_`).
+  - all spaces must be changed to underscores (`_`)
 - apostrophes (`'`)
 
 ### Article file names
 
-The name of an article file can be found in the `File Name` column of the [locales section](#locales). The location of a translation is to be placed in the appropriate English-named folder.
+The file name of an article can be found in the `File Name` column of the [locales section](#locales). The location of a translated article must be placed in the same folder as the English version.
 
 ### Index articles
 
-All folders should contain an article of some kind. If it seems impractical to create an article for a folder that holds other articles, create an index article instead. Index articles list off articles that are inside its own folder and could contain a description for each link.
+All folders should contain an article of some kind. If it seems impractical to create an article for a folder that holds other articles, create an index article instead. Index articles list off articles that are inside its own folder and may contain a description for each link.
 
 ### Disambiguation articles
 
-[Disambiguation](/wiki/Disambiguation) articles must be placed in the `/wiki/Disambiguation` folder.
+[Disambiguation](/wiki/Disambiguation) articles must be placed in the `/wiki/Disambiguation` folder. The main page must be updated to include the disambiguation article. Refer to [Disambiguation/Mod](/wiki/Disambiguation/Mod) as an example.
+
+Redirects must be updated to have the ambiguous keyword(s) redirect to the disambiguation article.
 
 ## Markdown
 
 All osu! wiki articles must adhere to the following rules.
 
 ### GitHub Flavored Markdown
 
-Parts of the GFM (GitHub Flavored Markdown) markup are supported in the osu! wiki. GFM has a small amount of markup that anyone can learn in about 3 minutes. However, do not expect a lot out of GFM as it is intended to be used as a bare minimum markup language.
+Some parts of the GitHub Flavored Markdown markup are supported in the osu! wiki.
 
 ### HTML
 
 <!-- ppy ruled HTML is not allowed. from:ppy#0001 during:2017-06-26 in:osu-wiki markdown -->
 
-**HTML use is prohibited!** Resturcture the formatting if you need to use HTML.
+HTML must not be used, with an exception for comments (see [comments](#comments)). The structure of the article must be redone if HTML is used.
 
 #### Comments
 
-HTML comments may be used for annotating text, but limit usage.
+HTML comments may be used for annotating text.
 
 ### Editing
 
-#### Escaping
+#### End of line sequence
 
-*Notice: The title of an article must not be escaped, as titles are parsed as plain text, not Markdown.*
+<!-- Parsing will break if it reads a file using CRLF. -->
```

Comment:
I don't think this is true, if you're referring to parsing on the website
not rly a problem though just a comment

Reply:
"Parsing" is broad, I could have worded that better. Probably something like: `<!-- YAML will display in the article on the website if the file is using CRLF -->`?